src/parts/_forms.scss

```diff
@@ -56,6 +54,36 @@ textarea:focus {
   box-shadow: 0 0 0 2px $focus;
 }
 
+input[type='checkbox'],
+input[type='radio'] {
+  width: 14px;
+  height: 14px;
+  display: inline-block;
+  vertical-align: middle;
+  margin: 0;
+  margin-right: 2px;
+}
+
+input[type='radio'] {
+  border-radius: 50%;
+}
+
+input[type='checkbox']:checked,
+input[type='radio']:checked {
+  background: $button-hover;
+
+  &::before {
+    content: '•';
+    display: block;
+    transform: translateY(-50%) translateY(-0.3px) translateX(-2.7px);
```

Comment:
I think in the past I've used something like this to center something exactly, but I'm not sure how well it works for characters. It would be worth a try?
```suggestion
    position: absolute;
    left: 50%; 
    top: 50%;
    transform: translateX(-50%) translateY(-50%);
```

The `position: absolute` and `top/left: 50%` center the top left of the bounding box. This `50%` uses the size of the parent container.

Then, the `translateX/translateY(-50%)` moves it relative to the width/height of the actual content. This should place it exactly centered in the circle.

This StackOverflow question also has some great suggestions:
https://stackoverflow.com/questions/20167074/how-to-center-a-single-character-both-vertically-and-horizontally-in-a-square-di

Reply:
*Dies inside* Let's try your solution - for some reason it didn't work for me in my testing earlier. I probably messed something up

Reply:
This did not seem to work - the characters don't appear at all

![image](https://user-images.githubusercontent.com/42556441/56462587-a6b91300-6393-11e9-9d3e-373ed466517f.png)

Looks like they're ending up in the middle of the page; I think we could fix this with a `position: relative` on the parent element

![image](https://user-images.githubusercontent.com/42556441/56462590-b9334c80-6393-11e9-9142-6647019d37c6.png)

Reply:
Relative positioning worked but it's still off

![image](https://user-images.githubusercontent.com/42556441/56462626-7faf1100-6394-11e9-9084-b9e044004d1b.png)

Reply:
Progress! Who knew that centering a dot could be this hard...

It looks like the actual box containing the character is centered, but the character isn't in the exact middle of a line:
![image](https://user-images.githubusercontent.com/6742479/56463273-b16cab80-6385-11e9-8e46-0c9dae735351.png)

`line-height: 100%` seems to help... a bit:
![image](https://user-images.githubusercontent.com/6742479/56463289-fee91880-6385-11e9-8f05-bd067ceadbb2.png)

This is the closest I got!
```
input[type=checkbox]:checked:before, input[type=radio]:checked:before {
    content: "•";
    display: block;
    position: absolute;
    left: 50%;
    top: 50%;
    transform: translateX(-50%) translateY(-50%) translateY(-0.5px);
    line-height: 100%;
}
```

![image](https://user-images.githubusercontent.com/6742479/56463329-bed66580-6386-11e9-9752-ce31fb304a4a.png)

...this all breaks if we ever change the root font size though. 🤷‍♂️ Maybe an SVG circle would be better?

Reply:
Yeah, I wasn't able to get anywhere either :grimacing: 

I also noticed that the dot seems to be in a different place depending on the zoom level as well. We could try SVG I guess - but what problems would that fix? We'd still have to center a graphic

Reply:
I think an SVG would have a more reasonable bounding box - right now, we're trying to center the entire line height of a character. The reason we're having trouble centering it is because the character isn't in the vertical middle of a line:
![image](https://user-images.githubusercontent.com/6742479/56463454-50df6d80-6389-11e9-8d0e-bccf59d10add.png)

I think an SVG would have a bounding box containing exactly the circle and not act like a character.

Reply:
@kognise  We could probably merge as is just to get this working. I don't think we need pixel perfect accuracy and we can always go back and fix it later. 😛